Tests/WorkflowPostProcess.Test.ps1

```diff
@@ -25,4 +25,30 @@ Describe "WorkflowPostProcess Action Tests" {
 
     # Call action
 
+    It 'Test DateTime serialization is locale-agnostic' {
+        # Simulate what WorkflowInitialize does - serialize a datetime in ISO 8601 format

```

Comment:
The whole test kind of just reimplements the logic from WorkflowPostprocess. Shouldn't we create a function in WorkflowPostprocess that we can call from here so we're testing the logic that actually runs in AL-Go workflows.